css/style.css

```diff
@@ -109,16 +110,27 @@ button {
   font-size: 44px;
   line-height: 1.27;
   text-align: center;
+  text-transform: uppercase;
+}
+
+.main {
+  margin-top: 108px;
 }
 
 /********************* Header styles *****************************/
 
-header {
-  margin: 30px 0;
+.header {
+  /* padding: 30px auto; */
   display: flex;
-  flex-direction: row;
   justify-content: space-between;
   align-items: center;
+  background-color: rgb(4, 11, 17);
+  position: fixed;
+  top: 0px;
+  left: 0;
+  width: 100%;
+  height: 108px;
```

Comment:
You can move this value to variable and use it here and as `margin-top` on main block

Reply:
Do you mean 

`:root {
        --indent-var: 108px;
   }


.main {
  margin-top: var(--indent-var);
}`.      ?

Reply:
#fixed